test/e2e/policies.test-suite/topology-aware/n4c16/test09-container-exit/code.var.sh

```diff
@@ -9,7 +9,8 @@ pyexec 'assert "pod0c0" in allocations'
 out '### Crash and restart pod0c0'
 vm-command "kubectl get pods pod0"
 vm-command "kill -KILL \$(pgrep -f pod0c0)"
-sleep 2
+vm-command 'kubectl wait --for=condition=Ready pods/pod0'
```

Comment:
In my VM, `kubectl wait --for=condition=Ready pods/pod0` seems to return with `pod/pod0 condition met` right after killing the process, because Kubernetes has not yet noticed that there is something wrong with the pod. Maybe a sleep would be useful after all to make this wait do something. Or just drop the wait.

Reply:
Hmm, ok. I did not notice any difference whether the sleep was there or not, but could be because of difference in host/VM combination. I can add the sleep back just in case. The wait-for is nice to have there too, if the startup takes more than 2sec.